### DISCUSSION TITLE
Labels for one language only / Automatically translated label

### DISCUSSION THREAD

**Discussion by bastianbeischer** [WeblateOrg/weblate#11020] [Q&A]
It seems when I apply a label to a string weblate will automatically apply that label to all languages.

I want to stick a label to strings only in one specific language. In particular, I want to undo a mistake: I have added a new language and used the Add-on to automatically translate all strings using machine translation. After that I had accidentally removed the "Automatically translated" label from some strings. I'm pretty sure that at that point that label was applying only to the specific string in that language. But now when I manually reapply the label it will show in all languages.

Is there any way to fix this apart from removing the lang and redoing the MT again?

**Comment by nijel** [comment]
There is no way to manually add label to a single language only.
